dask/dataframe/dispatch.py

```diff
@@ -0,0 +1,78 @@
+"""
+Dispatch in dask.dataframe.
+
+Also see extension.py
```

Comment:
Nitpick: Should this be `backends.py`?

Reply:
I think it this is right. There are some `Dispatch` objects initialized in there as well. I thought they were different in some way and should stay in extension.py, but I can't remember why I thought that. Do you think they should move into this file?

Reply:
Ah, I see. Since those are specifically for registering extension array, let's keep those where they are